docs/sources/installation/centos.md

```diff
@@ -6,153 +6,171 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
 
 Docker is supported on the following versions of CentOS:
 
-- [*CentOS 7 (64-bit)*](#installing-docker-centos-7)
-- [*CentOS 6.5 (64-bit)*](#installing-docker-centos-6.5) or later
+* CentOS 7.X 
+* CentOS 6.5 or higher 
 
-These instructions are likely work for other binary compatible EL6/EL7 distributions
-such as Scientific Linux, but they haven't been tested.
+Installation on other binary compatible EL6/EL7 distributions such as Scientific
+Linux might succeed, but Docker does not test or support Docker on these
+distributions.
 
-Please note that due to the current Docker limitations, Docker is able to
-run only on the **64 bit** architecture.
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using CentOS-managed packages, consult your
+CentOS documentation.
 
-## Kernel support
+##Prerequisites
 
-Currently the CentOS project will only support Docker when running on kernels
-shipped by the distribution. There are kernel changes which will cause issues
-if one decides to step outside that box and run non-distribution kernel packages.
+Docker requires a 64-bit installation regardless of your CentOS version. 
+Docker requires that your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does not. We make an exception for CentOS 6.5. To run Docker on [CentOS-6.5](http://www.centos.org) or later, you need kernel 2.6.32-431 or higher. 
```

Comment:
(just a nit) Both sentences above start with "Docker requires ...". Perhaps add some variation.

Do we need to keep the mention that only the kernel shipped with the distribution is supported? (I've seen people trying other kernels)